site/content/api/Device-Connection-API.md

```diff
@@ -0,0 +1,138 @@
+| *subject*    | yes       | *properties*             | *string*  | MUST be set to `set-last-gw`. |
+| *gateway_id* | yes       | *application-properties* | *string*  | The identifier of the gateway that last acted on behalf of the device identified by the *device_id* property. If a device connects directly instead of through a gateway, the device's identifier MUST be specified here. |
+
+The body of the message SHOULD be empty and will be ignored if it is not.
+
+**Response Message Format**
+
+A response to a *set last known gateway for device* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}).
+
+The response message's *status* property may contain the following codes:
+
+| Code | Description |
+| :--- | :---------- |
+| *204* | OK, the last known gateway for the device was updated. |
+| *404* | Not Found, there is no device registered with the given *device_id* within the given *tenant_id*, or no gateway registered with the given *gateway_id*. |
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+### Get Last Known Gateway for Device
+
+Clients use this command to *retrieve* the gateway that last acted on behalf of a given device.
+
+**Message Flow**
+
+
+The following sequence diagram illustrates the flow of messages involved in a *Client* retrieving the last known gateway for a Device.
+
+![Get Last Known Gateway for Device message flow](../get-last-gw_Success.png)
+
+
+**Request Message Format**
+
+The following table provides an overview of the properties a client needs to set on a message to retrieve the last known gateway for a device in addition to the [Standard Request Properties]({{< relref "#standard-request-properties" >}}).
+
+| Name        | Mandatory | Location                 | AMQP Type | Description |
+| :---------- | :-------: | :----------------------- | :-------- | :---------- |
+| *subject*   | yes       | *properties*             | *string*  | MUST be set to `get-last-gw`. |
+
+The body of the message SHOULD be empty and will be ignored if it is not.
+
+**Response Message Format**
+
+A response to a *get last known gateway for device* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}).
+
+The response message includes payload with a JSON object having the following properties:
+
+| Name             | Mandatory | JSON Type     | Description |
+| :--------------- | :-------: | :------------ | :---------- |
+| *gateway-id*     | *yes*     | *string*      | The ID of the last known gateway for the device, or the device ID itself if the device doesn't support usage via a gateway. |
+| *update-date*    | *no*      | *string*      | The date that the information about the last known gateway for the device was last updated. The date is formatted in ISO-8601 format, e.g. '2011-12-03T10:15:30Z'. |
```

Comment:
we should provide a link to the specification of  the format here. See https://www.eclipse.org/hono/api/tenant-api/#data-volume-configuration-format

Reply:
done